docs/source/installation.rst

```diff
@@ -2,19 +2,213 @@
 Installation
 ============
 
-BentoML is distributed as a Python package available `on PyPI <https://pypi.org/project/bentoml/>`_.
+Install 🍱 BentoML with your favourite package manager, alongside with whichever deep learning
+library you are working with, and you are ready to go!
+
+🍱 BentoML is distributed as a Python Package available `on PyPI <https://pypi.org/project/bentoml/>`_.
 
 * BentoML supports Linux/UNIX, Windows, and MacOS.
 * BentoML requires Python 3.7 or above.
 
+.. code-block::
+
+   pip install --user bentoml
```

Comment:
What does `--user` do? Can it be further simplified to just `pip install bentoml`?

Reply:
If people are using `pip` as global python interpreter, `--user` ensure it doesn't install bentoml to global pip

Reply:
I'm in favor of changing this to `pip install bentoml`. Does anybody else specifically suggest `--user`?

Reply:
I'd recommend that we leave the advanced options out. Exactly how user wants to install is situational. We mainly want to talk users that bentoml is pip installable.

Reply:
https://pip.pypa.io/en/stable/user_guide/#user-installs